Site/bc_weather_station_dashboard/website/models.py

```diff
@@ -57,4 +83,60 @@ class WeatherStation(models.Model):
     SOLAR_RADIATION_CM3 = models.FloatField()
 
     def __str__(self):
-        return self.STATION_NAME
\ No newline at end of file
+        return self.STATION_NAME
+    
+# The Dashboard model is used to store user type information for dashboard order showing station data
+class Dashboard(models.Model):
+    LAYOUT_CHOICES = [
+        ('firefighter', 'Firefighter Layout'),
+        ('common', 'Common Layout'),
+    ]
+    
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    theme = models.CharField(max_length=200, default='default')
+    layout = models.CharField(max_length=200, choices=LAYOUT_CHOICES, default='common')
+    
+    def __str__(self):
+        return self.user.username + "'s Dashboard Preferences"
+
+class Alert(models.Model):
```

Comment:
May need to change depending on how we decide to implement alerts, for now, looks good